ecommerce/frontend/src/utils.js

```diff
@@ -70,6 +70,29 @@ export function sleep(ms) {
     return new Promise(resolve => setTimeout(resolve, ms))
 }
 
+/**
+ * Data formatter
+ * A single credit card info from backend -> props of Cards component from react-credit-cards
+ */
+export function formatCreditCard({
+    id,
+    name,
+    owner_name,
+    serial_number,
+    expiration_date: {
+        month,
+        year
+    },
+    cvc
+}) {
+    return {
+        name: owner_name,
+        number: serial_number,
+        expiry: (month <= 9 ? `0${month}` : month) + '/' + (year % 1000),
```

Comment:
for month two comments:
 - sometimes january is counted as 0 sometimes 1, make sure of that
 - use `month.toString().padLeft(2, '0')` instead 

shouldn't year be year % 100 instead?